app/models/app_config.rb

```diff
@@ -30,4 +30,11 @@ class AppConfig < ApplicationRecord
   # List of launched features still guarded by a flag.
   # Use the features rake tasks for editing this key
   LAUNCHED_FEATURES = 'launched_features'.freeze
+  # Default subsample for special biohub samples.
+  BIOHUB_DEFAULT_SUBSAMPLE = 'biohub_default_subsample'.freeze
+  # Default max_input_fragments for special biohub samples.
+  BIOHUB_DEFAULT_MAX_INPUT_FRAGMENTS = 'biohub_default_max_input_fragments'.freeze
+  # The special biohub projects to apply the above two defaults to.
+  # Stored as a JSON array of project_ids.
+  BIOHUB_PROJECT_IDS = 'biohub_project_ids'.freeze
```

Comment:
We could generalize this concept to `WHITELIST_PROJECT_IDS`. It doesn't look like there is any biohub specific logic, only the name.

Reply:
Good point. I've generalized it to SUBSAMPLE_WHITELIST_PROJECT_IDS.